scheduler/classes.c

```diff
@@ -688,7 +688,7 @@ cupsdSaveAllClasses(void)
   strftime(temp, sizeof(temp) - 1, "%Y-%m-%d %H:%M", &curdate);
```

Comment:
Since this is no longer used, should it (and related) be removed?  (There are a few other similar.)

Reply:
Probably, yes. Surprised the code scanning tools didn't flag this as set but not used...

Reply:
[bug-fix-rollup-1 811e6226b] Eliminate unused variables.